### ISSUE TITLE
Textbox: does not support keyup, keydown or keypress event

### ISSUE THREAD

**Issue by kmlhossen** [eBay/ebayui-core#612]
For accessibility, we have to listen to `Enter` key event on textbox, but currently ebay-textbox does not have any associated events for keyup, keydown or keypress.

An explicit event like `textbox-focus` for keyup, keydown or keypress will be helpful.

**Comment by ianmcburnie** [comment]
@kmlhossen If you are blocked by this, how about listening to form submit event? That should fire on ENTER if textbox is placed inside of form. Still though, we should expose the events that you mention. Should be simple enough.

**Comment by ianmcburnie** [comment]
Punting to v2.3.0

**Comment by kmlhossen** [comment]
@ianmcburnie  differentiating between entering to submit the form and enter to got the next screen is getting a bit clumsy for us. 

It will be really great if we can give these events. What is the timeline for v2.3.0? 

can we do it early if its a small fix from your side?

**Comment by ianmcburnie** [comment]
We should be able to get them out in v2.2.0 for early next week, or as an early prerelease of v2.3.0 for mid next week.

Still though, an ENTER key press in a textbox should always do something (i.e. proceed forwards with their task), otherwise it feels like broken behavioiur to users.

**Comment by ianmcburnie** [comment]
@kmlhossen Will just `keydown` suffice for your needs? Keyup and keypress are used a lot less frequently.

**Comment by kmlhossen** [comment]
@ianmcburnie  yes keydown will work as I need to know user clicked Enter or not

**Comment by ianmcburnie** [comment]
Merged. This will be available in 2.2.0.
